org.moneymanagerex.MMEX.yml

```diff
@@ -68,6 +68,10 @@ modules:
     buildsystem: cmake
     config-opts:
       - -DCMAKE_BUILD_TYPE=Release
+    build-options:
+      arch:
+        x86_64:
+          cflags: -DCMAKE_C_FLAGS=''
```

Comment:
This should be in `config-opts:` above however resetting build flags to null value isn't good solution.  The whole `set (CMAKE_C_FLAGS "")` line should be patched out.